go/libraries/doltcore/remotesrv/grpc.go

```diff
@@ -383,15 +383,6 @@ func (rs *RemoteChunkStore) Rebase(ctx context.Context, req *remotesapi.RebaseRe
 		return nil, status.Error(codes.Internal, "Could not get chunkstore")
 	}
 
-	logger.Printf("found %s", repoPath)
-
-	err = cs.Rebase(ctx)
```

Comment:
ChunkStore state is centrally coordinated through the server process so we don't need to rebase `cs` here?
(also clients are using a getRoot call anyway)